src/views/PrivacyPolicy.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <section class="section">
+    <div class="container">
+      <router-link to="/dashboard">&larr; zurück</router-link>
+      <h2 class="subtitle is-2 has-text-weight-light">Datenschutzhinweise</h2>
```

Comment:
I tried viewing this on a mobile device and noticed that the browser is not line-breaking this word properly.
Maybe some `&shy;` markers can improve this. :slightly_smiling_face: